audits/manifest/icons-192.js

```diff
@@ -30,7 +28,7 @@ class ManifestIcons192 {
 
   static audit(inputs) {
     let hasIcons = false;
-    const manifest = manifestParser(inputs.manifest).value;
+    const manifest = inputs.manifest;
```

Comment:
you're going to want the uglier check I suggested here. e.g. if they have one icon for several sizes and set `"sizes": "96x96 192x192 256x256"` this check will count that as invalid